Compiler/src/ssair/verify.jl

```diff
@@ -67,7 +67,10 @@ function check_op(ir::IRCode, domtree::DomTree, @nospecialize(op), use_bb::Int,
             imported_binding = partition_restriction(bpart)::Core.Binding
             bpart = lookup_binding_partition(min_world(ir.valid_worlds), imported_binding)
         end
-        if !is_defined_const_binding(binding_kind(bpart)) || (bpart.max_world < max_world(ir.valid_worlds))
+        if (!is_defined_const_binding(binding_kind(bpart)) || (bpart.max_world < max_world(ir.valid_worlds))) &&
+                (op.mod !== Core) && (op.mod !== Base)
+            # Core and Base are excluded because the frontend uses them for intrinsics, etc.
```

Comment:
Does this mean the verifier (and also somewhat the IR semantics) are also broken for any `topmod` other than Base (like `Compiler`)? I don't think the frontend directly references `Base` anywhere (although I know there are plenty of references to Core)

Reply:
Yes. This was a little bit of a band-aid since we didn't move these globalrefs out into statement position. We need to take a decision on the semantics of these, but I want to punt that to a later PR